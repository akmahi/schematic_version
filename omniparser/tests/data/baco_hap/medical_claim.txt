107274487100    2487124     100621931   05FORTIN                             Test1          F19980808D 48183          M2022101920221019J0140                                                                                                                                                              MINUTECLINIC DIAGNOSTICS OF INDIANA LLC   PGRP  N203207688460771800      FORD MOTOR COMPANY - HOURLY ACTIVE      0000000000000139.00202210192022101900000000000001.0J0140                         3074F       0000000000000000.000000000000000000.000000000000000000.000000000000000000.000000000000000000.00AAS00109100000396175                              0020000000000     0000000000000000.0000HC3         100000133872        100000133872        1000HHMR0001  10000011 1000           0211   U314L41  L41  L41                                     CPT4 1000 W OOK ST                                          Zionsville     IN 0000000000000000.00E0000000000000000.000000000000000000.00    NP    0000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000.00        000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001932171634NNOO107274487100HP  ICD-10                        
107274487100    2487124     100621931   05FORTIN                             Test1          F19980808D 48183          M2022101920221019J0140                                                                                                                                                              MINUTECLINIC DIAGNOSTICS OF INDIANA LLC   PGRP  N203207688460771800      FORD MOTOR COMPANY - HOURLY ACTIVE      0000000000000139.00202210192022101900000000000001.0J0140                         99203       0000000000000139.000000000000000000.000000000000000000.000000000000000000.000000000000000000.00AAS00109100000396175                              0010000000000     0000000000000000.0000HC3         100000133872        100000133872        1000HHMR0001  10000011 1000           0211   G224L29  L29  L29                                     CPT4 1000 W OAK ST                                          Zionsville     IN 0000000000000000.00E0000000000000000.000000000000000000.00    NP    0000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000.00        000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001932171634YNOO107274487100HP  ICD-10                        
107273780600    2521696     100031148   02GADWELL                            Test12         F20030922D 48180          M2022101820221018R490                          Z6824                         R059                          M940                                                                     PROGRESSIVE HEALTH CARE PC                PRAC  N461377771481801847      FORD MOTOR COMPANY - HOURLY ACTIVE      0000000000000206.00202210182022101800000000000001.0M940                          81002       0000000000000006.000000000000000003.980000000000000000.000000000000000000.000000000000000003.98AAS00109100000346304                              0030076801421     0000000000000000.0000HC3         100000022483        100000327299        1000HHMR0001  10000011 1000           0211   K133E12  E12  E12                             20221102CPT4 100011 ECORSE RD                                        TAYLOR         MI 0000000000000000.00E0000000000000000.000000000000000000.00    IM    0000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000.00        000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001780696229NNOI107273780600HP  ICD-10                        
107273031600    1823740     100182012   04LYNCH                              Test           M19990215S 48235          M2022092020220920Z20828                                                                                                                                                             MULTIPLAN, INC                            PGRP  N133068979100031004      FORD MOTOR COMPANY - HOURLY ACTIVE      0000000000000024.37202209202022092000000000000001.0Z20828                                    0000000000000024.370000000000000024.370000000000000000.000000000000000000.000000000000000024.37AAS00109100000343038                              0010076801045     0000000000000000.0000HC3         100000000069        100000000069        1000HHMR0001  10000011 1000           0211   INV           002                             20221102     100 FIFTH AVENUE                                       NEW YORK       NY 0000000000000000.00B0000000000000000.000000000000000000.00    CCTR  0000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000.00        000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000009999999999NNOO107273031600HP  ICD-10                        
107281522300    3143153     100147023   00PATTISON                           Test121        M19610624M 48166          M2022102420221024M25552                        R262                          Z471                          Z96642                                                                   TEAM REHABILITATION SERVICES LLC MI       PGRP  Y412064132480731501      FORD MOTOR COMPANY - HOURLY ACTIVE      0000000000000475.00202210242022102400000000000001.0M25552                        97140GP     0000000000000105.000000000000000015.000000000000000000.000000000000000000.000000000000000015.00AAS00109100000155720                              0020003902781     0000000000000000.0000HC3         100000399881        100000399881        1000HHMR0001  10000011 1000           0211   P1C1E14  E14  E14                             20221102CPT4 10000 W 14 MILE RD                                      ROYAL OAK      MI 0000000000000000.00E0000000000000000.000000000000000000.00    PM    0000000000000000.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000.00        000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001548932619NNOI107281522300HP  ICD-10                        
