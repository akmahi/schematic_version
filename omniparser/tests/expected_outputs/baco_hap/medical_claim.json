[
	{
		"claim": {
			"id": "107274487100",
			"line_amount_paid": "0000000000000000.00",
			"line_coinsurance_amount": "0000000000000000.00",
			"line_copay_amount": "0000000000000000.00",
			"line_deduct_amount": "0000000000000000.00",
			"line_number": "002",
			"line_place_of_service": "11",
			"line_primary_diagnosis_code": "J0140",
			"line_service_end_date": "20221019",
			"line_service_start_date": "20221019",
			"line_total_charges": "0000000000000000.00",
			"line_type_of_service": "U314",
			"patient": {
				"address": {
					"zip": "48183"
				},
				"dob": "19980808",
				"first_name": "Test1",
				"gender": "F",
				"last_name": "FORTIN",
				"relationship_code": "D",
				"subscriber_id": "100621931"
			},
			"service_end_date": "20221019",
			"service_start_date": "20221019",
			"total_amount": "0000000000000139.00",
			"type": "M"
		}
	},
	{
		"claim": {
			"id": "107274487100",
			"line_amount_paid": "0000000000000000.00",
			"line_coinsurance_amount": "0000000000000000.00",
			"line_copay_amount": "0000000000000000.00",
			"line_deduct_amount": "0000000000000000.00",
			"line_number": "001",
			"line_place_of_service": "11",
			"line_primary_diagnosis_code": "J0140",
			"line_service_end_date": "20221019",
			"line_service_start_date": "20221019",
			"line_total_charges": "0000000000000139.00",
			"line_type_of_service": "G224",
			"patient": {
				"address": {
					"zip": "48183"
				},
				"dob": "19980808",
				"first_name": "Test1",
				"gender": "F",
				"last_name": "FORTIN",
				"relationship_code": "D",
				"subscriber_id": "100621931"
			},
			"service_end_date": "20221019",
			"service_start_date": "20221019",
			"total_amount": "0000000000000139.00",
			"type": "M"
		}
	},
	{
		"claim": {
			"id": "107273780600",
			"line_amount_paid": "0000000000000003.98",
			"line_coinsurance_amount": "0000000000000000.0",
			"line_copay_amount": "0000000000000000.00",
			"line_deduct_amount": "0000000000000000.00",
			"line_diagnosis_code_1": "Z6824",
			"line_diagnosis_code_2": "R059",
			"line_diagnosis_code_3": "M940",
			"line_number": "003",
			"line_place_of_service": "11",
			"line_primary_diagnosis_code": "R490",
			"line_service_end_date": "20221018",
			"line_service_start_date": "20221018",
			"line_total_charges": "0000000000000006.00",
			"line_type_of_service": "K133",
			"patient": {
				"address": {
					"zip": "48180"
				},
				"dob": "20030922",
				"first_name": "Test12",
				"gender": "F",
				"last_name": "GADWELL",
				"relationship_code": "D",
				"subscriber_id": "100031148"
			},
			"service_end_date": "20221018",
			"service_start_date": "20221018",
			"total_amount": "0000000000000206.00",
			"type": "M"
		}
	},
	{
		"claim": {
			"id": "107273031600",
			"line_amount_paid": "0000000000000024.37",
			"line_coinsurance_amount": "0000000000000000.00",
			"line_copay_amount": "0000000000000000.00",
			"line_deduct_amount": "0000000000000000.00",
			"line_number": "001",
			"line_place_of_service": "11",
			"line_primary_diagnosis_code": "Z20828",
			"line_service_end_date": "20220920",
			"line_service_start_date": "20220920",
			"line_total_charges": "0000000000000024.37",
			"line_type_of_service": "INV",
			"patient": {
				"address": {
					"zip": "48235"
				},
				"dob": "19990215",
				"first_name": "Test",
				"gender": "M",
				"last_name": "LYNCH",
				"relationship_code": "S",
				"subscriber_id": "100182012"
			},
			"service_end_date": "20220920",
			"service_start_date": "20220920",
			"total_amount": "0000000000000024.37",
			"type": "M"
		}
	},
	{
		"claim": {
			"id": "107281522300",
			"line_amount_paid": "0000000000000015.00",
			"line_coinsurance_amount": "0000000000000000.0",
			"line_copay_amount": "0000000000000000.00",
			"line_deduct_amount": "0000000000000000.00",
			"line_diagnosis_code_1": "R262",
			"line_diagnosis_code_2": "Z471",
			"line_diagnosis_code_3": "Z96642",
			"line_number": "002",
			"line_place_of_service": "11",
			"line_primary_diagnosis_code": "M25552",
			"line_service_end_date": "20221024",
			"line_service_start_date": "20221024",
			"line_total_charges": "0000000000000105.00",
			"line_type_of_service": "P1C1",
			"patient": {
				"address": {
					"zip": "48166"
				},
				"dob": "19610624",
				"first_name": "Test121",
				"gender": "M",
				"last_name": "PATTISON",
				"relationship_code": "M",
				"subscriber_id": "100147023"
			},
			"service_end_date": "20221024",
			"service_start_date": "20221024",
			"total_amount": "0000000000000475.00",
			"type": "M"
		}
	}
]
